pavelib/servers.py

```diff
@@ -27,10 +31,12 @@ def run_server(system, settings=None, port=None, skip_assets=False, contracts=Fa
 
     if not settings:
         settings = DEFAULT_SETTINGS
+        asset_settings = settings
```

Comment:
I removed that line and all the tests carried on passing, so I think it isn't necessary.